app/src/main/java/org/fossasia/phimpme/gallery/util/CustomTabService.java

```diff
@@ -8,7 +8,6 @@
 import android.support.customtabs.CustomTabsServiceConnection;
 import android.support.customtabs.CustomTabsSession;
 import android.widget.Toast;
-
```

Comment:
Don't remove this, it was part of the intendation so that we could seperate packages provided by the system from our ones.

Reply:
Yeah, I understood after seeing many other files. Thanks :blush: